packages/primitives/radio/meta.tsx

```diff
@@ -1,35 +1,79 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { TComponentConfig } from 'autoprops'
 import { component, startWithType } from 'refun'
+import { Block } from '@primitives/block'
+import { Text } from '@primitives/text'
 import { RadioInput, RadioGroup, TRadioInput, TRadioGroup } from './src'
+import { RadioContext } from './src/context'
+
+// This Component will be passed by the consumer,
+// but without it, it's quite pointless to have
+// RadioInput on Sandbox.
+type TPresentationProps = {
+  value: string,
+  isDisabled?: boolean,
+}
+
+const PresentationElement = ({ isDisabled, value }: TPresentationProps) => (
+  <RadioContext.Consumer>
+    {({ groupValue }) => (
+      <Block style={{
```

Comment:
So this is only for testing in Meta, right?

Reply:
correct. 👍